### ISSUE TITLE
Control inputs resolution

### ISSUE THREAD

**Issue by bhack** [z-x-yang/Segment-and-Track-Anything#19]
Can we control SAM/Deaotl final input resolution from the UX/params?

Can we enable Gradio zooming in the UX?

**Comment by yamy-cheng** [comment]
In the next version, we will implement the feature for users to control the resolution. You can indirectly achieve Gradio component scaling by using the zoom function in your browser.

**Comment by bhack** [comment]
![immagine](https://user-images.githubusercontent.com/1710528/234595607-b3191da2-a62a-4434-b259-9c29b3b6acbe.png)

Do you meant that you can click fine fingers like in our official tutorial screenshot just with zooming the browser?

**Comment by yamy-cheng** [comment]
> ![immagine](https://user-images.githubusercontent.com/1710528/234595607-b3191da2-a62a-4434-b259-9c29b3b6acbe.png)
> 
> Do you meant that you can click fine fingers like in our official tutorial screenshot just with zooming the browser?

Yes, all the examples in our official tutorial are reproducible.

**Comment by bhack** [comment]
No I meant that the segmented fingers seems to be cut. Are you able to fix the prompt just zooming the browser? At it seem hard to click on that small detail.

**Comment by yamy-cheng** [comment]
> No I meant that the segmented fingers seems to be cut. Are you able to fix the prompt just zooming the browser? It seem hard to click on these small details.

I'm sorry for misunderstanding your meaning. The ability of SAMTrack to segment small details is constrained by SAM. We will add the ability for users to paint to complete masks to our to-do list. This feature may fulfill your needs.

**Comment by bhack** [comment]
> The ability of SAMTrack to segment small details is constrained by SAM. 

Yes it is why we don't know how much click/stoke are needed and could need to zoom it.

If I remember correctly the R50 deaotl encoder/decoder conf (1024px) has enough resolution to propagate eventually some fine details right?
